### ISSUE TITLE
Mermaid diagram not rendering in my Sphinx Doc w/ Myst Parser

### ISSUE THREAD

**Issue by solarslurpi** [executablebooks/sphinx-design#132]
### Describe the bug

**context**
```
pip install sphinxcontrib-mermaid
```
add to extensions list in `conf.py`:
```
extensions = [
    ...,
    'sphinxcontrib.mermaid'
]

open an existing md file, e.g.: `index.md` and add:
```{mermaid}

    sequenceDiagram
      participant Alice
      participant Bob
      Alice->John: Hello John, how are you?
```

running sphinx autodoc so pages are rebuilt.  Note: at one point I thought I'd clean and refresh so started with `make clean` - didn't matter.

**expectation**
I expected to get a really wow looking diagram.

**bug**
But instead got a line of text with no Mermaid "js goo".. I didn't get an error (that I could find). I thought it might be due to not downloading a .js file but???
```
sequenceDiagram participant Alice participant Bob Alice->John: Hello John, how are you?
```

### Reproduce the bug

I tried very hard to follow the directions the 10 different times I tried to get this working.  I apologize if I cannot figure this out.  I have had that problem before.

### List your environment

[pip_list.txt](https://github.com/executablebooks/sphinx-design/files/11820271/pip_list.txt)

Python 3.9.2

[systeminfo.txt](https://github.com/executablebooks/sphinx-design/files/11820346/systeminfo.txt)

**Comment by chrisjsewell** [comment]
Heya, I'm not sure how this issue ended up here as it's not to do with sphinx-design, you might to open it on https://github.com/mgaitan/sphinxcontrib-mermaid
